src/components/root.js

```diff
@@ -1,8 +1,8 @@
 import ActionButton from './action-button';
+import styleFormatter from '../utils/style-formatter';
 
-export function teardown() {}
-
-export function render(element, props) {
+export default function render(element, props) {
   const button = ActionButton({ element, ...props });
   element.appendChild(button);
+  styleFormatter.setFontSizeAndFamily({ button, ...props });
```

Comment:
Why do we have to call it here and not inside ActionButton?

Reply:
So the issue is that we need the button to be appended to the element so we can do our calculations on the width and height of the button (containing borders). If we don't have the button appended the size will be 0 and all calculations will be incorrect. This will be okay once the button is rendered but not on first render when you open up the page (or do a refresh)

Reply:
Ok 👍